framework/wazuh/core/cluster/master.py

```diff
@@ -666,6 +666,10 @@ async def send_entire_agent_groups_information(self):
         start_time = get_utc_now()
         logger.info('Starting.')
 
+        # Recalculate group hashes before retrieving agent groups info
+        logger.debug('Recalculating agent-group hash.')
+        await AsyncWazuhDBConnection().run_wdb_command(command='global recalculate-agent-group-hashes')
```

Comment:
It seems that the `recalculate-agent-group-hashes` could return an error code:
https://github.com/wazuh/wazuh/blob/22449c835383c96f7aa5a1eb65cc1837a990dccc/src/wazuh_db/wdb_parser.c#L6034

Which would raise a WazuhInternalError  inside the `run_wdb_command`:
https://github.com/wazuh/wazuh/blob/22449c835383c96f7aa5a1eb65cc1837a990dccc/framework/wazuh/core/wdb.py#L121-L122

Or a `WazuhError`:
https://github.com/wazuh/wazuh/blob/22449c835383c96f7aa5a1eb65cc1837a990dccc/framework/wazuh/core/wdb.py#L244-L245

We should catch said error, log it and continue with the task.